mindjunkies/lecture/tests/test_forms.py

```
@@ -0,0 +1,287 @@
+            ["Only video files (.mp4, .avi, .mov, .mkv) are allowed."],
+        )
+
+    def test_missing_video_file(self):
+        """Test form with missing video_file"""
+        data = {"video_title": "Test Video"}
+        form = LectureVideoForm(data=data)
+        self.assertTrue(form.is_valid())  # video_file is not required
+        video = form.save(commit=False)
+        video.lecture = self.lecture
+        video.save()
+        self.assertEqual(video.video_title, "Test Video")
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)